src/effects/TiltShift2.tsx

```diff
@@ -49,22 +64,24 @@ const TiltShiftShader = {
 
 export class TiltShiftEffect extends Effect {
   constructor({
-    blendFunction = BlendFunction.ADD,
-    blur = 20.0,
-    gradient = 1100.0,
-    start = [0.5, 0.0],
-    end = [0.5, 1.0],
-    delta = [1, 1],
+    blendFunction = BlendFunction.Normal,
+    blur = 1.0, // [0, 1], can go beyond 1 for extra
+    taper = 0.5, // [0, 1], can go beyond 1 for extra
+    start = [0.5, 0.0], // [0,1] percentage x,y of screenspace
+    end = [0.5, 1.0], // [0,1] percentage x,y of screenspace
+    sampleCount = 20.0, // number of blur samples
```

Comment:
20 texture reads per frag is a lot imo, could we use a lower default? like 10? imo the noise is photographic, it doesn't look half bad.

Reply:
@drcmda sure, just updated default to 10 in bc271f5e61071abe1295de6d927bf3308cc19669